Submodels/output_handling.py

```
@@ -1,11 +1,446 @@
+        Parameters
+        ----------
+        - analysis_name : str
+            A string of the analysis name. Must equal the filename extension used for walls.xxx, tflow.xxx, tdat.xxx, boundary_layer.xxx, and flowfield.xxx. 
+
+        Returns
+        -------
+        None
+        """ 
+
+        # Simple input validation
+        if analysis_name == None:
```

Ruff (0.8.2)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)